docs/reference/browser-support.md

```diff
@@ -7,7 +7,10 @@ slug: /browser-support
 
 # Browser Support
 
-Stencil builds Web Components that run natively or near-natively in all widely used desktop and mobile browsers.
+Stencil builds Web Components that run natively in all widely used desktop and mobile browsers.
```

Comment:
This page looks something like this now:
![Screenshot 2023-06-14 at 3 55 26 PM](https://github.com/ionic-team/stencil-site/assets/1930213/515b2742-aa0d-4b62-86ce-6d74f1f62721)

I wonder if it makes sense to roll this into our support policy (https://stenciljs.com/docs/support-policy) and just setup a redirect. The page is kinda bare and doesn't really warrant its own route anymore (IMO). Thoughts?

Reply:
it's kind of a stub now, I'm down with merging 👍

Reply:
Yeah, I looked at this this morning with a fresh set of eyes - I think merging makes the most sense to me